rotation.py

```diff
@@ -147,7 +156,19 @@ def __init__(self, no_of_qubit: int, rotation_amount: Fraction) -> None:
     def __str__(self) -> str:
         return '{}: {}'.format(self.rotation_amount, self.ops_list) 
             
-        
+
+    def to_latex(self) -> str:
+        return_str = super().to_latex() 
+        return_str += '_\\frac{'
+        if abs(self.rotation_amount.numerator) != 1:
+            return_str += str(self.rotation_amount.numerator)
+        elif self.rotation_amount.numerator == -1:
+            return_str += '-'
```

Comment:
Is there a particular reason why we are adding the minus sign inside the numerator? like it messes the layout otherwise? Normally you would have the minus sign in front of the fraction.

Reply:
if the numerator is -1/1 i want it to display as -pi/pi on the graph.

Reply:
I think this does what you want while keeping the minus sign out of the fraction
```
from fractions import Fraction

def render_phase_frac_to_latex(phi : Fraction):
  """Assumes phi is multiplied by pi"""
  if phi.numerator== 0: return "0"

  sign = "" if phi.numerator > 0 else "-"
  num = "" if abs(phi.numerator) == 1 else str(abs(phi.numerator))

  if phi.denominator == 1:
    return "%s%s\\pi"%(sign,num)

  den = str(phi.denominator)

  return "%s\\frac{%s\\pi, %s}"%(sign,num,den)

```
Example:
```
if __name__ == "__main__":
  print(render_phase_frac_to_latex(Fraction(0,1)))
  print(render_phase_frac_to_latex(Fraction(1,1)))
  print(render_phase_frac_to_latex(Fraction(-1,1)))
  print(render_phase_frac_to_latex(Fraction(1,2)))
  print(render_phase_frac_to_latex(Fraction(-1,2)))
  print(render_phase_frac_to_latex(Fraction(1,4)))
  print(render_phase_frac_to_latex(Fraction(-1,4)))
  print(render_phase_frac_to_latex(Fraction(1,8)))
  print(render_phase_frac_to_latex(Fraction(-1,8)))
  print(render_phase_frac_to_latex(Fraction(7,8)))
  print(render_phase_frac_to_latex(Fraction(-7,8)))
```
Outputs
```
0
\pi
-\pi
\frac{\pi, 2}
-\frac{\pi, 2}
\frac{\pi, 4}
-\frac{\pi, 4}
\frac{\pi, 8}
-\frac{\pi, 8}
\frac{7\pi, 8}
-\frac{7\pi, 8}
```

Reply:
implemented in 6e4fe5f.